pkg/utils/get-repository-info.go

```diff
@@ -0,0 +1,57 @@
+package utils
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+// GetRepoUrl - Gets the URl of a git repo.
+func GetRepoUrl() (string, error) {
+	gitLocation, err := exec.LookPath("git")
+
+	if err != nil {
+		return "", fmt.Errorf("unable to find git on system: %w", err)
+	}
+
+	cmd := exec.Command(gitLocation, "config", "--get", "remote.origin.url")
+
+	cmdOutput, err := cmd.CombinedOutput()
+
+	if err != nil {
+		return "", err
+	}
+
+	url := ParseGitUrl(strings.TrimSuffix(string(cmdOutput), "\n"))
+
+	return url, nil
+}
+
+//ParseGitUrl - Parses a git url to ensure that it is in HTTPS format
+func ParseGitUrl(url string) string {
+	if strings.Contains(url, "git@") {
+		url = strings.Replace(url, ":", "/", 1)
+		url = strings.Replace(url, "git@", "https://", 1)
+		url = strings.Replace(url, ".git", "", 1)
+		return url
+	}
+	return url
+}
```

Comment:
I don't think any of this is necessary (and might be github-specific, breaking other providers)